sources/packages/backend/apps/api/src/route-controllers/application/application.controller.service.ts

```diff
@@ -209,6 +210,26 @@ export class ApplicationControllerService {
     return details;
   }
 
+  /**
+   * Transformation util for application header information for AEST DTO.
+   * @param applicationDetail
```

Comment:
param missing description

Reply:
This method is removed and similar method is used for `ApplicationHeaderTitle` component.